bin/plpbridge

```diff
@@ -263,9 +271,17 @@ elif args.chip is not None:
 else:
     raise Exception('A chip or a config file must be specified')
 
+# Need to flash the flash image and boot from flash
+if args.boot_mode == "jtag_hyper":
+    binaries= [os.environ["INSTALL_DIR"] + "/bin/flasher"]
+# Need to flash the flash image and still boot from jtag
+elif args.fimages != []:
+    binaries= [os.environ["INSTALL_DIR"] + "/bin/flasher"] + args.binaries
+# boot from jtag
+else:
+    binaries = args.binaries
 
-
-bridge = db.get_bridge(config=config, verbose=args.verbose, binaries=args.binaries)
+bridge = db.get_bridge(config=config, verbose=args.verbose, binaries=binaries, fimages=args.fimages)
```

Comment:
The prototype of this function should not change, you should pass the flash image argument direcly from the flash command